ui/prometheus-plugin/src/plugins/PrometheusDatasourceEditor/types.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { RequestHeaders } from '@perses-dev/core';
+
+// TODO unify this one with PrometheusDatasourceSpec from prometheus-datasource.tsx ? or rename one or the other for disambiguition?
+export interface PrometheusDatasourceSpec {
```

Comment:
Can you tell me again the difference between the two? You told me already but I forgot. This SpecFull thingy is a terrible name indeed :smile: I'm wondering if we can't find a more meaningful name.

Reply:
I'm not crazy about the `PrometheusDatasourceSpecFull` alias name either but I don't have a great solution on how these types could be refactored right now